Haaretz/htz-frontend
# feat(haaretz.co.il): Set up an Apollo based app

This updates the `haaretz.co.il` app to use Apollo for data fetching. It includes a demonstration of rendering slots based on the JSON page data. The goal is to discuss and iterate on this quickly so we can make sure this is a suitable approach you'll be happy with.

The GraphQL engine runs completely in the browser.

To test it:

```console
$ git checkout wip/apollo-based-store
$ yarn run bootstrap
$ cd packages/apps/haaretz.co.il
$ yarn run dev
```

You may notice that it takes longer than it should to load a second page. That's because in dev mode, none of the page bundles are prebuilt – it's running webpack just-in-time to serve your request. In production mode, they're all prebuilt:

```console
$ yarn run build
$ yarn run start
```
